### DISCUSSION TITLE
deployment not found

### DISCUSSION THREAD

**Discussion by BASHIR219** [vercel/community#4699] [Help]
### Summary

I had upload node js project on vercel, after deployment I am getting error on url. Eg [NOT_FOUND] Code: NOT_FOUND

### Example

https://mafecbackend.vercel.app/

### Steps to Reproduce

I am not getting this part

**Comment by amyegan** [comment]
Hi @BASHIR219. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to help with debugging?

It would also be helpful to know what steps you took to deploy the project and what build settings are were used.
